backend/app/assets/stylesheets/spree/backend/globals/_variables.scss

```diff
@@ -206,3 +207,14 @@ $z-index-navbar-flyout:          1000 !default;
 // Sidebar
 //--------------------------------------------------------------
 $border-sidebar:                 1px solid $color-sidebar-border !default;
+
+// Media queries
+//--------------------------------------------------------------
+$desktop-width: 959px !default;
+$tablet-width: 768px !default;
+$mobile-width: 480px !default;
+
+$large-device: "only screen and (min-width: #{$tablet-width})";
+$desktop: "only screen and (max-width: #{$desktop-width})";
+$tablet: "only screen and (max-width: #{$tablet-width})";
+$mobile: "only screen and (max-width: #{$mobile-width})";
```

Comment:
Yeah, but Bootstrap is already part of Solidus, so all mixins it has should be available.

- https://v4-alpha.getbootstrap.com/layout/overview/#responsive-breakpoints

Try the above ones, they should work:

```
@include media-breakpoint-up(xl) { ... }
```